src/libslic3r/GCode.cpp

```diff
@@ -4557,11 +4557,15 @@ std::string GCode::extrude_loop(ExtrusionLoop loop, std::string description, dou
         loop.split_at(last_pos, false);
 
     const auto seam_scarf_type = m_config.seam_slope_type.value;
-    const bool enable_seam_slope = ((seam_scarf_type == SeamScarfType::External && !is_hole) || seam_scarf_type == SeamScarfType::All) &&
+    bool enable_seam_slope = ((seam_scarf_type == SeamScarfType::External && !is_hole) || seam_scarf_type == SeamScarfType::All) &&
         !m_config.spiral_mode &&
         (loop.role() == erExternalPerimeter || (loop.role() == erPerimeter && m_config.seam_slope_inner_walls)) &&
         layer_id() > 0;
 
+    if (enable_seam_slope && m_config.seam_slope_conditional.value) {
+        enable_seam_slope = enable_seam_slope &&
```

Comment:
`enable_seam_slope` is always `true` when reaching this place, so there is no need to write `enable_seam_slope &&`.

Reply:
hehe, good catch

Reply:
changed